packages/nova-lib/lib/server/meteor_patch.js

```diff
@@ -0,0 +1,98 @@
+import { Meteor } from 'meteor/meteor';
+import { WebApp } from 'meteor/webapp';
+
+import { Utils } from '../modules/index.js';
+
+// our bindEnvironment keep the original function and auto rename the return function
+export const bindEnvironment = (fn) => {
```

Comment:
yes, i know that. 
i explain why i use these.
custom bindEnvironment i use for keep the original function, also  for get/set their function name for debug to see how many webapp.connectHandlers.stack item on it.
and
another custom webAppConnectHandlers, the improvements is i can add webapp connectHandler with a order number, so finally i can run it by order.
why i write this, because inject-data-middleware must put it to the last,
and i placed inject-data to the nova:lib for other package can use it.
if i don't patch webapp with order feature,
inject-data can not placed in nova:lib
and finally other packages can't use it.
and then i can't remove meteorhacks_inject-initial.
yes, i accept your opinion, we must minimize. and i know i wrote too many code on it.
may be we can think another way to do something like this